### DISCUSSION TITLE
An issue after booting

### DISCUSSION THREAD

**Discussion by heikebotaru** [CobanRamo/Lenovo-IdeaPad-S340-15IIL#2] [Q&A]
Hello. Thanks for your great work. I am doing Hackintosh with your EFI as a reference. I am using an Ideapad-S340-14IIL (i5-1035G1). Thanks to your EFI, most of the functions are fine. However, after booting, the screen goes dark and I have to wait for a while before I can use it. When the screen appears, it's like waking up from sleep. Do you have any advice for me? Thank you very much.

**Comment by CobanRamo** [comment]
Hello heikebotaru,
I have the i7-1065G7, so with the IceLake's it is unfortunately normal as far as I have noticed so far.
This problem for a few seconds has not yet been solved with the IceLake's.

Greetings Coban

**Reply by heikebotaru** [reply]
Hello. CobanRamo,
Thank you for your reply. It seems to take a long time to solve the problem. In my current situation, the darkened screen often does not come back. When I press the volume button I get a beep, so I think everything is working except the screen. As a test, I set the time to screen sleep to the shortest possible time, which increased the chance of the screen recovering. This may be related to the screen sleep function.
Thanks

**Reply by CobanRamo** [reply]
it is not in the sleep state, this has something to do with the initialized driver.

Try the buttons 1,2,3,4 etc. in this black screen state.
for me this process accelerates.

What have you set in the Bios DVMT? which graphic ID do you use? Can you make your Config.plist available?

Greetings Coban

**Reply by heikebotaru** [reply]
Thank you for the information. 

My settings are as follows
DVMT = 160M in the BIOS
graphic ID = 5C8A0000.

I'll put config.plist up. 
[config.plist.zip](https://github.com/CobanRamo/Lenovo-IdeaPad-S340-15IIL/files/6265604/config.plist.zip)

Thanks lots.

**Reply by CobanRamo** [reply]
You can still test with DVMT 64MB, 160 & 64 work very well for me.

Have you also tested the other IDs?
https://github.com/acidanthera/WhateverGreen/blob/master/Manual/FAQ.IntelHD.en.md
![IceLake Native ID´s](https://user-images.githubusercontent.com/68192365/113787832-099c4400-973c-11eb-9d78-513b00abaaab.png)

**Reply by CobanRamo** [reply]
I also recommend that you clean up your boot arguments. If you have the values in the properties, you don't need any boot arguments.

See WhatEverGreen boot arguments ...
https://github.com/acidanthera/WhateverGreen

![Properties](https://user-images.githubusercontent.com/68192365/113787875-1de04100-973c-11eb-8eae-09b1a18be0e8.png)

**Comment by CobanRamo** [comment]
If you want to enter the Windows & Linux manually you have to enter YOUR path, you have taken it from me, it cannot work for you ...

![Windows   Linux Entries](https://user-images.githubusercontent.com/68192365/113788403-22592980-973d-11eb-8122-dcc67f87a6d2.png)

With the OC Configurator Tool you can easily determine the path.
![oc configurator](https://user-images.githubusercontent.com/68192365/113788632-ba571300-973d-11eb-9558-cffa7a9f73d7.png)

**Reply by heikebotaru** [reply]
Hi CobanRamo

I followed your instructions. I set the DVMT to 64M and the device-ID to 0x8A5C. The darkening of the screen at the time of starting does not change, but I think that the screen display afterwards became somewhat smooth. Every time I got informations that I thought might be useful in various places, I added them, so it became impossible for me to organize them. I have been working with a very limited understanding of OC, so your instructions have been a very good learning experience for me. Thank you very much for the very useful informations you have given me.

Greetings heikebotaru
